React/Views/RCTTabBar.m

```diff
@@ -105,6 +105,9 @@ - (void)reactBridgeDidFinishTransaction
     _tabController.viewControllers = viewControllers;
     _tabsChanged = NO;
   }
+  
+  [[UITabBarItem appearance] setTitleTextAttributes:[NSDictionary dictionaryWithObjectsAndKeys:_unselectedTintColor, NSForegroundColorAttributeName, nil] forState:UIControlStateNormal];
+  [[UITabBarItem appearance] setTitleTextAttributes:[NSDictionary dictionaryWithObjectsAndKeys:self.tintColor, NSForegroundColorAttributeName, nil] forState:UIControlStateSelected];
 
```

Comment:
This is a global appearance attribute that will affect all tab bar items, right? Not just the ones in this tab bar.

I appreciate that it's unlikely that an app would have more than one tab bar, but I think we should find a better solution.

Ideally the RCTTabBar would apply these appearance attributes just to its own tab bar items.